tests/app/test_jinja_filters.py

```diff
@@ -455,3 +375,63 @@ def test_format_address_fields_with_uprn():
     assert (
         get_formatted_address(address_fields) == "7 Evelyn Street<br>Barry<br>CF63 4JG"
     )
+
+
+@pytest.mark.parametrize(
+    "min_or_max_value, expected_width",
+    [
+        (None, 10),
+        (1, 1),
+        (123123123123, 20),
+    ],
+)
+def test_numeric_detail_answer_input_class_config(
+    answer_schema_number, min_or_max_value, expected_width
+):
+    if min_or_max_value:
+        answer_schema_number["maximum"] = {"value": min_or_max_value}
```

Comment:
If `min_or_max_value` is always `maximum`, then it should be named `max_value`.

Reply:
Done